### DISCUSSION TITLE
Welcome to nix-on-droid android App discussions!

### DISCUSSION THREAD

**Discussion by t184256** [nix-community/nix-on-droid-app#4] [Announcements]
Please stick to just discussing the nix-on-droid Android app side, not what runs inside it. Note that I have no capacity or Android expertise to work on issues. PRs are welcome though.

**Comment by phanirithvij** [comment]
@t184256 by chance do you have a shell.nix/flake.nix setup to build the apk?

**Comment by t184256** [comment]
I don't, such contributions would be welcome.
